### PR TITLE
Fixes #1793 and fixes #1792

### LINKED ISSUE #1792: No error symbol for broken layer
App version: 4.6.1
            Build date: Wed Jun 10 11:57:32 AEST 2015
            Build number: 15
            Git ref: 1d2478cc16914fdfc8b7ade8abc245331f0e96fe

Steps to reproduce:

Add a collection that does not load

What does happen:

User doesn't really see anything happen.

What should happen:

See red error triangle

### LINKED ISSUE #1793: No loading symbol for collections
App version: 4.6.1
            Build date: Wed Jun 10 11:57:32 AEST 2015
            Build number: 15
            Git ref: 1d2478cc16914fdfc8b7ade8abc245331f0e96fe

Steps to reproduce:

Add a collection to the map

What should happen:

User sees a loading symbol so they know it is loading

What does happen:

No loading symbol is seen anywhere

### PR DESCRIPTION
**Pull request by pmbohm**
Toolbar items using FontAwesome. The step 1 add extra criteria button is now only visible when available instead of being disabled. It was hard to tell when the extra criteria button were active or disabled as there are no expand/collapse icons anymore on step1, which matches Step2

### CODE REVIEW

File: web-app/js/portal/lang/en.js [hunk 1]
```diff
@@ -179,7 +179,7 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
     resetActionText: 'reset',
 
     // FacetFilterPanel
-    addAnother: 'Add another filter',
+    addAnother: 'Add another filter - where possible',
```
Comment (dnahodil) [inline web-app/js/portal/lang/en.js]: Is it really necessary to say 'where possible' as we now only show the icon when it *is* possible?

**Comment by dnahodil** [comment]
Looks pretty good @pmbohm. Just the one comment.

File: web-app/js/portal/lang/en.js (hunk 1 above)
Comment (pmbohm) [inline web-app/js/portal/lang/en.js]: @dnahodil Agh yes 
If someone see's this tooltip, (when possible ! hahaa) they will then have an explanation as to why the link disappears sometimes.

**Comment by dnahodil** [comment]
@pmbohm Okay, I'm happy to merge then. Looks like it needs a rebase first, though.

**Comment by pmbohm** [comment]
@dnahodil it passed Travis now :)

**Pull request merged**